src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -35,9 +34,19 @@ trait ClassLikeSupport:
       signatureOnly: Boolean = false,
       modifiers: Seq[Modifier] = classDef.symbol.getExtraModifiers(),
     ): DClass = 
-      val supertypes = getSupertypes(classDef).map{ case (symbol, tpe) =>
-        LinkToType(tpe.dokkaType.asSignature, symbol.dri, kindForClasslike(symbol))
+      def hackGetParents(classDef: ClassDef): Option[List[Tree]] = scala.util.Try(classDef.parents).toOption
+
+      def getSupertypesGraph(classDef: ClassDef, link: LinkToType): Seq[(LinkToType, LinkToType)] = {
+        val smbl = classDef.symbol
+        val parents = if smbl.exists then hackGetParents(smbl.tree.asInstanceOf[ClassDef]) else None
```

Comment:
I don't understand this code. Why do we need to take the symbol of a tree and then to take the tree of that symbol? That should be a no-op. In what cases does it matter?

Reply:
I just realized I was misintepreting all the time relation between `Tree` and `ClassDef`. Will change